content/en/docs/started/installing-kubeflow.md

```diff
@@ -224,9 +362,8 @@ If you need support, please consider using a [packaged distribution](#packaged-d
 Nevertheless, we welcome contributions and bug reports very much.
 {{% /alert %}}
 
-<a id="next-steps"></a>
```

Comment:
```
{{% alert title="Warning" color="warning" %}}
Kubeflow is a complex system with many components and dependencies.
Using the Kustomize manifests requires some understanding of Kubernetes, Istio, and Kubeflow itself.
The Kubeflow community support for Kustomize manifests is best-effort for environment-specific issues or custom configurations.
Nevertheless, we welcome contributions and bug reports very much.
{{% /alert %}}
```

@andreyvelich i was not able to make a direct code suggestion here so I am just pasting the full alert.